pyupgrade.py

```diff
@@ -1186,15 +1254,23 @@ def _eq(test, n):  # type: (ast.Compare, int) -> bool
         )
 
     @staticmethod
-    def _compare_to_3(test, op):
-        return (
-            isinstance(test.ops[0], op) and
-            isinstance(test.comparators[0], ast.Tuple) and
-            len(test.comparators[0].elts) >= 1 and
-            all(isinstance(n, ast.Num) for n in test.comparators[0].elts) and
-            test.comparators[0].elts[0].n == 3 and
-            all(n.n == 0 for n in test.comparators[0].elts[1:])
-        )
+    def _compare_to_3(
+        test,  # type: ast.Compare
+        op,  # type: Union[Type[ast.cmpop], Tuple[Type[ast.cmpop], ...]]
+    ):
+        # type: (...) -> bool
+        if not (
+                isinstance(test.ops[0], op) and
+                isinstance(test.comparators[0], ast.Tuple) and
+                len(test.comparators[0].elts) >= 1 and
+                all(isinstance(n, ast.Num) for n in test.comparators[0].elts)
+        ):
+            return False
+
+        # checked above but mypy needs help
+        elts = cast("List[ast.Num]", test.comparators[0].elts)
```

Comment:
heh, I guess I don't have `double-quote-string-fixer` pre-commit hook enforced do I 🤔

Reply:
I opened another PR for that